generate.js

```diff
@@ -4,6 +4,7 @@ const ncp = require('ncp').ncp
 const orgs = require('./out/data.json')
 
 const datetime = new Date(fs.statSync('./out/data.json').mtime).toUTCString()
+const rootURL = process.env.DEPLOY_PRIME_URL || process.env.URL
```

Comment:
`DEPLOY_PRIME_URL` always contains the branch name :/ e.g. `http://master--gci-leaders.netlify.com`.
That is not ideal.

Use only `URL`.